recipes/django-maintenance-mode/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "django-maintenance-mode" %}
+{% set version = "0.15.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django-maintenance-mode-{{ version }}.tar.gz
+  sha256: f8c500b077045db5444d4d4eb613fd1085b51f960d6d61b13b1cce461e1d0102
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - django >=1.7
+
+test:
+  imports:
+    - maintenance_mode
+    - maintenance_mode.management
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/fabiocaccamo/django-maintenance-mode
+  summary: django-maintenance-mode shows a 503 error page when maintenance-mode is on.
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
Since the license file is manually packaged, I submitted a PR upstream to hopefully deal with this in future releases: https://github.com/fabiocaccamo/django-maintenance-mode/pull/69

Could you please put a comment above `license_file: LICENSE.txt` that reads:

```yaml
# License file manually packaged. See: https://github.com/fabiocaccamo/django-maintenance-mode/pull/69
license_file: LICENSE.txt
...
```

This will be a visual reminder to check if the license file is in the source distribution in future releases when the feedstock gets updated.

Reply:
add...
Thanks @synapticarbors!